menyer/menyer/emmies-restaurang/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="sv">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>emmies restaurang</title>
+    <link rel="stylesheet" href="style.css">
+</head>b
+
+<body>
+    <main>
+        <h1>emmies restaurang</h1>
+        <section id="starters">
+            <h2>Förrätter</h2>
+            <div class="food-item">
+                <img src="https://i.imgur.com/Ba7MqyA.jpg" width="400" alt="Langos">
+            </div>
+            <p>
+                <strong>Langos</strong> - <i>Langos med creme fraishe, ost och skinka</i> 
+            </p>
+            <p class="food-item-price"> <strong>39 kr</strong> </p>
+            <div class="food-item"></div>
```

Comment:
Empty div should proberbly be removed. It has an effect on your design that I don't think you want. 
![image](https://user-images.githubusercontent.com/3958329/94987337-1611cb00-0566-11eb-805e-39b3798479c2.png)